### PR TITLE
BOO-522 Updated for rally-sbt-plugin

### PR DESCRIPTION
**Pull request by patrick-bohan**
@jeffmay @nicksovich cause no one else has touched this repo

### CODE REVIEW

**Comment by jeffmay** [comment]
retest this please

**Comment by jeffmay** [comment]
LGTM

**Comment by patrick-bohan** [comment]
Is broken due to dependency, will be fixed by the scalacheck-ops PR

File: build.sbt [hunk 1]
```diff
@@ -75,7 +38,7 @@ lazy val playJsonTests = project in file("playJsonTests") settings(common: _*) s
     "org.scalacheck" %% "scalacheck" % "1.12.2",
     "org.scalatest" %% "scalatest" % "2.2.4",
     "com.rallyhealth" %% "scalacheck-ops" % "1.0.0"
-  ).map(_.withSources())
```
Comment (nicksovich) [inline build.sbt]: woohoo!

**Comment by nicksovich** [comment]
LGTM

**Comment by patrick-bohan** [comment]
retest this please

**Comment by patrick-bohan** [comment]
Release v0.3.0 http://jenkins.infra.careverge.com:8080/job/Release_play-json-ops/3/

**Pull request merged**